### PR TITLE
Resolved Issue #43 - Update Icons for Labels and Subscribers

### LINKED ISSUE #43: 🤔 [ISSUE] - Update Icons for Labels and Subscribers
### Description

The current icons for labels and subscribers need to be updated for a more modern and consistent user interface. The new icons should align with the overall design and provide clearer representation for users.

image
image

### Reproduction steps

```bash
1. Go to Labels & Subscribers panels
```

### Logs

_No response_

### Browsers

_No response_

### OS

_No response_

### PR DESCRIPTION
**Pull request by JaiSharma1710**
# Motivation
This PR solves the issue mentioned in #43 raised by @Emnaghz 

Fixes # (issue)
Fixes #43 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [X] I have performed a self-review of my own code

# Final icons images

Screenshot from 2024-09-22 07-50-02

Screenshot from 2024-09-22 07-50-11

### CODE REVIEW

File: frontend/src/components/labels/index.tsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ export const Labels = () => {
           if (deleteDialogCtl?.data) deleteLabel(deleteDialogCtl.data);
         }}
       />
-      <PageHeader icon={FolderIcon} title={t("title.labels")}>
+      <PageHeader icon={LabelIcon} title={t("title.labels")}>
```
Comment (marrouchi) [inline frontend/src/components/labels/index.tsx:161]: Can you please use the same icon used in the sidebar ?
image

**Review by marrouchi — CHANGES_REQUESTED** [review]
Thank you for your contribution @JaiSharma1710 🙏

File: frontend/src/components/labels/index.tsx (hunk 1 above)
Comment (JaiSharma1710) [reply frontend/src/components/labels/index.tsx:161]: yhe sure

Comment (JaiSharma1710) [reply frontend/src/components/labels/index.tsx]: @marrouchi updated the pr

Screenshot from 2024-09-22 12-42-57

**Review by yassinedorbozgithub — APPROVED** [review]
LGTM :+1:

**Review by marrouchi — APPROVED** [review]
Awesome @JaiSharma1710 👏  thanks !

**Pull request merged**